docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -1,326 +1,752 @@
 
-* add your settings in `{config-file}`
-* annotate your entities with `@Entity` and make them extend `PanacheEntity`
-* place your entity logic in static methods in your entities
+The JPA `@entity` annotation is at the heart of Hibernate ORM. When applied to a Java class, it maps the class to a specific database table. The fields in the class map to database columns. Each instance of the class represents a database row.
 
-Follow the link:hibernate-orm#setting-up-and-configuring-hibernate-orm[Hibernate set-up guide for all configuration].
+== Hibernate ORM with the {ProductName} Panache JPA extension
 
-In your `pom.xml`, add the following dependencies:
+Hibernate ORM is the most widely used JPA implementation. It provides all of the features of an ORM. However, for a simple implementation with the most common data mappings you can use the {ProductName} Panache JPA extension to write more concise code.
 
-* the Panache JPA extension
-* your JDBC driver extension (`quarkus-jdbc-postgresql`, `quarkus-jdbc-h2`, `quarkus-jdbc-mariadb`, ...)
+The Hibernate ORM with Panache extension contains the following classes:
 
-[source,xml]
+* `PanacheEntity`
+* `PanacheEntityBase`
+* `PanacheQuery`
+* `PanacheRepository`
+* `PanacheRepositoryBase`
+* `PanacheEntity`
+
+The {ProductName} Panache JPA extension eliminates the following Hiberante ORM issues:
+
+* *Duplicate ID logic*
++
+You use the `PanacheEntity` class to create entities. Because the ID is defined automatically in the `PanacheEntity` class, you do not need to define it in your `PanacheEntity` instance. However, you can specify your own ID in your instance if you prefer.  If you require a custom ID strategy, you can extend the `PanacheEntityBase` class instead.
+
+* *Useless `getter` and `setter` functions*
++
+Because Java does not support properties, when you write Java code you must create fields and then generate `getter` and `setter` functions for those fields, even if all you need to do is read or write to the field. With Panache, if you define a field as a public field and then read or write to that field, Panache generates the required `getter` and `setter` functions. In addition, you can write other accessors as required.
+* *Entity definitions split from operations*
++
+In the traditional Java EE model, you put the the entity definition in one class, the model, and the operations performed on the model in another class, the data access object (DAO) or repository. The Panache active record method enables you to create the entity definition and the repository in the same class.
+
+* *Verbose Hibernate queries*
++
+Panache provides Hibernate Query Language (HQL) shortcuts to simplify your queries. You do not write parts of the query that you do not need.
+
```

Comment:
We probably miss what was ill-explained in the source guide as `Hibernate is very general-purpose, but does not make it trivial to do trivial operations that make up 90% of our model usage` which actually refers to the number of predefined methods such as find/list/delete etc that we provide. This is an important point, even though it was previously not properly explained (in the current guide), which is probably why you omitted it.